components/ProgressView/README.md

```diff
@@ -187,3 +190,20 @@ tracking its development:
 
 - Task: [Implement a color themer](https://www.pivotaltracker.com/story/show/157095443)
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+### MDCProgressView
+
+#### -accessibilityValue
+
+Like UIProgressView, MDCProgressView's `accessibilityValue` is based on the current value of the ProgressView's
+`progress` property. UIProgressView has internal logic that makes it so that a `progress` value of .476 will yield an
+`accessibilityValue` of "fifty percent". To ensure that the MDCProgressView follows the same behavior, the
+MDCProgressView class has a static UIProgressView that instances query for its `accessibilityValue` whenever
+they need to provide their own.
```

Comment:
Who or what is "they" in this sentence?

Reply:
The instances of MDCProgressView. Whoops, unclear!